### ISSUE TITLE
TLOD not scaling up above 1000 ft

### ISSUE THREAD

**Issue by PixelPascal** [ResetXPDR/MSFS2020_AutoFPS#23]
Hello,

I installed tested the auto tlod for msfs. It is scaling down the TLOD to 50, but above 1000 ft or higher the tlod stucks at 50. It is not scaling up. What can be the cause and solution for it?

Thanks in advance!

Pascal

**Comment by ResetXPDR** [comment]
If your actual FPS is not getting higher than your target FPS then TLOD won't increase. If so, try setting a lower target FPS that is at least around 10% less than the best FPS you can achieve.

**Comment by PixelPascal** [comment]
Hello, Thank you for responding. That fixed it in a blink of a eye!Met vriendelijke groet, Pascal Theune | Pixel Designs Pixel Designs | Media CreationZenegroen 366841 KS ArnhemT 026 - 84 85 231I www.pixeldesigns.nlOp 6 mrt. 2024 om 11:59 heeft ResetXPDR ***@***.***> het volgende geschreven:﻿
If your actual FPS is not getting higher than your target FPS then TLOD won't increase. If so, try setting a lower target FPS that is at least around 10% less than the best FPS you can achieve.

**Comment by ResetXPDR** [comment]
Glad I could help. I will close this issue now that you have resolved it.

**Comment by PixelPascal** [comment]
Yes, of course! Thank you.Met vriendelijke groet, Pascal Theune | Pixel Designs Pixel Designs | Media CreationZenegroen 366841 KS ArnhemT 026 - 84 85 231I www.pixeldesigns.nlOp 6 mrt. 2024 om 13:57 heeft ResetXPDR ***@***.***> het volgende geschreven:﻿
Glad I could help. I will close this issue now that you have resolved it.
